test/verify/check-machines-dbus

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+import unittest
+
+from testlib import *
+import machineslib
+
+
+class TestMachinesDBus(machineslib.TestMachines):
+
+    def skipIfNotLibvirtDbusAvailable(self):
+        m = self.machine
+        try:
+            m.execute("busctl introspect org.libvirt /")
+        except:
+            raise unittest.SkipTest("libvirt-dbus is not available");
+
+    def allowSystemDBusUnprivileged(self):
+        m = self.machine
+        # Add admin user to libvirt group so that he has access to system libvirt-dbus
+        m.execute("sudo usermod -a -G libvirt admin")
```

Comment:
This looks much better indeed. Also, defining `m` for just a single use is overkill :-) (Same for the helper function above).

Reply:
Done